ui/partials/main/thActionButton.html

```diff
@@ -27,7 +27,7 @@
            href="https://secure.pub.build.mozilla.org/buildapi/self-serve/{{::repoName}}/rev/{{::resultset.revision}}">BuildAPI</a></li>
     <li><a target="_blank" data-ignore-job-clear-on-click
            href=""
-           ng-show="user.is_staff"
+           ng-show="user.is_staff && ['mozilla-inbound', 'autoland'].includes(currentRepo.name)"
```

Comment:
Could we store this list in a ``$scope`` var in the appropriate controller instead of inline here?  And please add a comment as to why we're only showing it for these particular repos.

Thanks!  :)